app/controllers/admin/content_controller.rb

```diff
@@ -140,6 +154,7 @@ def do_add_or_remove_fu
   def real_action_for(action); { 'add' => :<<, 'remove' => :delete}[action]; end
 
   def new_or_edit
+    @articles = Article.all
```

Comment:
This query could potentially be large, and only needed in the specific scenario of admin and edit. It would be good to conditionally do this so as not to create more queries unnecessarily